packages/ui/src/ui/pages/query-tracker/QueryResultsView/YQLTable/YQLTable.tsx

```diff
@@ -407,13 +453,20 @@ function unquote(value?: string) {
     return value;
 }
 
-const cellCopyToClipboardClassName = block('cell-copy-to-clipboard');
+const cellActionClassName = block('cell-action');
 
 interface TableCellProps {
     children?: StrictReactNode;
     rawValue?: string;
+    onPreviewClick: () => void;
+    isTruncated?: boolean;
+}
+
+function isRawValue(value: string | undefined): value is string {
```

Comment:
What is the reason to have such type-guard function? 
Isn't it much clear to use `typeof value === string`/`Boolean(value)` check in place of usage?